cloud-azure/src/test/resources/json/arm-network-dnszone.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
what is the difference between main and test file ?

Reply:
There is a test in AzureNetworkDnsZoneTemplateBuilderTest to compare the content of the generated and expected templates.